.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.7.0
+    working_directory: ~/nativeauthenticator
+    steps:
+      - checkout
+      - run: 
+          name: Installing dependencies
+          command: sudo pip install -r dev-requirements.txt
```

Comment:
You likely want to `pip install .` here as well. Sudo might not be required if you use `--user`

Reply:
To expand a little bit:

`pip install .` will install package in `setup.py` in current directory (`.`). Currently the error in CircleCI seems to be:

```
ModuleNotFoundError: No module named 'jupyterhub'
```

This is because the JupyterHub package is specified as part of install_requires in setup.py, but not in dev-requirements.txt. setup.py is the correct place for it, but we should use pip to run it before testing here.